### PR TITLE
armor fixes!

### PR DESCRIPTION
**Pull request by benj8560**
Fixes 2 armors not being space proof. As well as some minor balance changes 

The medical void armor printed at R&D is now space proof duh its a space suit. Helmet also given higher energy protection to make it closer to the armor its attached to. 
Suit given reduced slowdown given it's meant for medical use. Still rather slow but on the slightly higher end as far as voidsuits go for slowdown.

Psion heavy and light armors given insulated gloves given they are intended to be a upgrade from standard psion armor. Heavy armor had its pressure resistance fixed as well and is now space proof.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/void/station.dm [hunk 1]
```diff
@@ -599,7 +600,7 @@
 	armor_list = list(
 		melee = 25,
 		bullet = 30,
-		energy = 20,
+		energy = 40,
```
[the remark this answers is not in the document]
Comment (benj8560) [reply code/modules/clothing/spacesuits/void/station.dm:603]: the armor itself has 50...or 60 one of the two. That helmet prints along side the suit

[the remark this answers is not in the document]
Comment (cdb-is-not-good) [reply code/modules/clothing/spacesuits/void/station.dm:603]: I'd rather not buff this. It's a bit weird that the med void has 55 energy armor in the first place

[the remark this answers is not in the document]
Comment (benj8560) [reply code/modules/clothing/spacesuits/void/station.dm:603]: to be clear this is not the normal medical armor. As stated in the PR this is printed in R&D and costs plat

[the remark this answers is not in the document]
Comment (benj8560) [reply code/modules/clothing/spacesuits/void/station.dm:603]: this is not the voidsuits medical gets round start. This is ablative armored voidsuits they have to print and unlock

[the remark this answers is not in the document]
Comment (benj8560) [reply code/modules/clothing/spacesuits/void/station.dm:603]: Regardless I don't care about to much about the helmet protection it mostly just seemed strange as fuck to me for a ablative expensive suit to have a weirdy crap helmet. Mostly just making sure you don't think this is the normal med voidsuit.

[the remark this answers is not in the document]
Comment (benj8560) [reply code/modules/clothing/spacesuits/void/station.dm:603]: and onto another point. The standard medical voidsuits helmet has 35 energy. Why would this helmet intended to be a upgrade have worse energy protection for a suit made to protect from energy.

[the remark this answers is not in the document]
Comment (cdb-is-not-good) [reply code/modules/clothing/spacesuits/void/station.dm:603]: Oh? this is that weird suit that never gets made? I see.

[the remark this answers is not in the document]
Comment (benj8560) [reply code/modules/clothing/spacesuits/void/station.dm:603]: yesss never gets made cos the helmet was not space proof and it was the most heavy ass medic armor imaginable :P

**Pull request merged**